solar_installation/demo/sale_order_post.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <function model="sale.order" name="action_confirm">
+        <value eval="[
+            ref('sale_order_20'),
+            ref('sale_order_23'),
+            ref('sale_order_24'),
+            ref('sale_order_25'),
+            ref('sale_order_26'),
+            ref('sale_order_27'),
+        ]"/>
+    </function>
```

Comment:
```suggestion
    <function name="action_confirm" model="sale.order"
        eval="[
            ref('sale_order_20'),
            ref('sale_order_23'),
            ref('sale_order_24'),
            ref('sale_order_25'),
            ref('sale_order_26'),
            ref('sale_order_27'),
        ]"
    />
```
Try to see if this works and make the change for the whole file (it should work as it works in purchase_order_post.xml, if not then make it other records like this one so we maintain symmetry)
Also, make sure the name is before the model attribute

Reply:
made all the function records in symmetry